packs/github/actions/lib/base.py

```diff
@@ -35,12 +36,12 @@ def _web_session(self):
             'authenticity_token': token
         }
 
-        session_url = BASE_URL + session_path
+        session_url = WEB_URL + session_path
         session.post(session_url, data=login_data)
         return session
 
     def _get_analytics(self, category, repo):
-        url = 'https://github.com/' + repo + '/graphs/' + category + '.json'
+        url = WEB_URL + repo + '/graphs/' + category + '.json'
```

Comment:
Should we document that GetCloneStatsAction and GetTrafficStatsAction aren't supported for Github Enterprise? (My GHE v2.4.3 doesn't have a Traffic graph which require traffic-stats and clone-activity-data. [The docs](https://help.github.com/enterprise/2.5/user/articles/about-repository-graphs/) make it look like v2.5 doesn't either.)

Also, looks like this link might be broken for hosted Github now as well? Seems like my browser is requesting the normal URL (`/graphs/<category>`) but with `Content-Type: application/json`.